osquery/tables/system/darwin/homebrew_packages.cpp

```diff
@@ -62,6 +75,246 @@ std::vector<std::string> getHomebrewVersionsFromInfoPlistPath(
   return results;
 }
 
+// Looking for a "auto_updates true" line in a .rb file
+bool checkAutoUpdatesInRubyFile(const std::string& file_path) {
+  std::ifstream file(file_path);
+  if (!file.is_open()) {
+    TLOG << "Error opening file " << file_path;
+    return false;
+  }
+
+  std::string line;
+  std::regex auto_updates_regex(R"(^\s*auto_updates\s*true\s*$)");
```

Comment:
I wonder if there are cases where this isn't enough. This is a processed ruby file, after all. Still though, might be a start.

Reply:
I'm honestly not happy with the regex, but adding a full ruby parser seemed overkill for that purpose. Happy to revisit the decision if y'all think it would be better.